catt/controllers.py

```diff
@@ -54,15 +86,23 @@ def clear(self):
 
 
 class CastController:
-    def __init__(self):
+    def __init__(self, device_name):
         cache = Cache()
+        cached_ip = cache.get(device_name)
```

Comment:
Does this have to be the exact name? It would be nice if we could do a substring match "if name in chromecast_name".